### PR TITLE
Conner Johnson Checkpoint 01

### PR DESCRIPTION
**Pull request by cconnerj**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -23,7 +23,9 @@ aside{
   flex: 3;
 }
 .left, .right{
-  float:left;
+  display: flex;
+  flex-direction: row;
+  /* Fuzzy, will update */
```
Comment (cconnerj) [inline styles.css]: Thought you wanted flex property, 

add float: left;
delete: udates

File: index.html [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <!doctype html>
 <html>
   <head>
-    <link rel='stylesheet' href='styles.css'>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
```
Comment (jsm13) [inline index.html]: 👍

File: index.html [hunk 3]
```diff
@@ -1,7 +1,8 @@
...
+    <title> Hurray </title>
```
Comment (jsm13) [inline index.html]: 👍

File: index.html [hunk 4]
```diff
@@ -14,11 +15,11 @@
       <h2>The Main Content</h2>
       <div class='left'>
 	This is the left side
-	<img src='https://jesse.sh/img/me.jpg'>
+	<img src="https://jesse.sh/img/me.jpg"/>
```
Comment (jsm13) [inline index.html]: 👍 looking for `alt` but will count this (but keep in mind `/` doesn't have an effect, alt is important for screen readers and failures to load assets)

File: index.html [hunk 5]
```diff
@@ -14,11 +15,11 @@
...
       </div>
-      <div class='right'>
+      <nav class='right'>
```
Comment (jsm13) [inline index.html]: 👍

File: styles.css [hunk 6]
```diff
@@ -9,6 +9,7 @@ body{
 }
 header, footer, aside, .main{
   padding: 1em;
+  margin: 0;
```
Comment (jsm13) [inline styles.css]: 👍  wanted a new rule for direct children `header>*` but will count this

File: styles.css (hunk 1 above)
Comment (jsm13) [inline styles.css]: 👍 this question was odd. Sorry for the confusion

File: styles.css [hunk 7]
```diff
@@ -33,7 +36,16 @@ img{
...
+    order: 1;
+  }
+  main{
+    order: 2;
+  }
+  aside{
+    order: 3, 4;
+  }
+  footer{
+    order: 5;
```
Comment (jsm13) [inline styles.css]: 👍

File: workflow.txt [hunk 8]
```diff
@@ -1,10 +1,10 @@
...
+git push origin my-solution
 git fetch --all
-git add .
-git pull upstream master
 git merge my-solution
-fork on github
 git checkout master
-git push origin my-solution
+git pull upstream master
+git checkout -b my-solution
```
Comment (jsm13) [inline workflow.txt]: 👍

**Pull request closed without merging**